File: app/src/main/java/de/mchllngr/quickopen/base/BasePresenter.java
package de.mchllngr.quickopen.base;

import com.hannesdorfmann.mosby3.mvp.MvpBasePresenter;
import com.hannesdorfmann.mosby3.mvp.MvpPresenter;

/**
 * Base-class for work concerning every {@link MvpPresenter}.
 *
 * @param <V> view-interface for this fragment
 */
public class BasePresenter<V extends BaseView> extends MvpBasePresenter<V> {
}

File: app/src/main/java/de/mchllngr/quickero/base/BasePresenter.kt
package de.mchllngr.quickero.base

import com.hannesdorfmann.mosby3.mvp.MvpBasePresenter

/**
 * Base-class for work concerning every [MvpPresenter].
 *
 * @param <V> view-interface for this fragment
 */
open class BasePresenter<V : BaseView> : MvpBasePresenter<V>()
